CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.4)
 
-project(libappimage)
+project(libappimage VERSION 0.2.0)
+
+# more versioning
+set(libappimage_SOVERSION 0.2)
```

Comment:
I guess that there is no to declare an additional variable for the soversion. We can use
`PROJECT_VERSION_MAJOR` AND `PROJECT_VERSION MINOR` 

See https://cmake.org/cmake/help/latest/command/project.html

Reply:
This hasn't been resolved apparently. Please check again @kossebau @azubieta.

Reply:
We are to blame for doing the related discussion on irc :)
There I said:
> this completely depends on your strategy with the SOVERSION. most library projects I have seen use a single number and increase it on every non-backward-compatible ABI change, separate from the normal versioning (usually a semver one)
> for now I understood that TheAssassin wants "0.2" as SOVERSION for the 0.2 series, and once 1.x is reached, switch to a single number again, starting with "1"

And after checking that indeed the SONAME works with using "0.2" as SOVERSION appendix, @azubieta then completed with
> that's good
>we can merge that

But he might then want to do this here officially as well then :)

Reply:
Ah well, in that case... when you're not part of such a discussion, then it's hard to relate IRC and GitHub. Lights are green, let's merge this.